### ISSUE TITLE
Zombie Servers

### ISSUE THREAD

**Issue by tkennes** [contino/cloud-sustainability-dashboard#13]
Hola! 

Great start! So many ideas to take this further, why don't you start with zombie servers? E.g. VMs that are officially in use but have not shown any business activity for the last X months?

Tom

**Comment by laure-co** [comment]
Hi Tom,

Thank you for the feedback, it would be great to hear your ideas! We do show CPU Utilisation of EC2 and RDS instances over a configurable period of time which would would help identify zombie servers - would that be what you are looking for? Or perhaps this could be presented differently? Thank you.

**Comment by tkennes** [comment]
Hi,


I think it depends a little bit on the definition. Low CPU utilization could be an indication, but it might also just be overprovisioning. There has been some research from Koomey (the researcher behind Koomey's Law) about how many VMs  we're talking about, and some way's how to recognize them. The research is a bit older, but still getting referenced now and then: https://www.anthesisgroup.com/zombie-servers-hunting-down-the-lost-capital/

In short: focus on a lack of connectivity to the machine, are these metrics you are already collecting?


Tom
